### PR TITLE
gtk.lisp: set sandboxing in web-context's initialize-instance method

### PR DESCRIPTION
**Pull request by tiberious726**
Make sure this is taken care of by using a real constructor instead of
an ersatz one

### CODE REVIEW

**Comment by tiberious726** [comment]
Also test for actually having the feature in cl-webkit. tho I don't know if its absence should be a silent condition

**Comment by jmercouris** [comment]
Fantastic, thank you!

**Comment by jmercouris** [comment]
It's absence can be a silent condition, if anything we should just peg a version of cl Webkit to Nyxt rather than using feature flags

File: source/renderers/gtk.lisp [hunk 1]
```diff
@@ -75,10 +75,13 @@ data-manager will store the data separately for each buffer.")
   (:accessor-name-transformer (class*:make-name-transformer name)))
 (define-user-class buffer (gtk-buffer))
 
-(defmethod make-web-context ((manager webkit:webkit-website-data-manager))
-  (let ((context (make-instance 'webkit:webkit-web-context :website-data-manager manager)))
-    (webkit:webkit-web-context-set-sandbox-enabled context t)
-    context))
+(defclass webkit-web-context (webkit:webkit-web-context) ()
+  (:metaclass gobject:gobject-class))
```
Comment (Ambrevar) [inline source/renderers/gtk.lisp:79]: @tiberious726 Out of curiosity, why did you set the metaclass to `gobject-class`?

**Pull request merged**